base/libgit2/libgit2.jl

```diff
@@ -494,14 +494,14 @@ a `GitError`. This is roughly equivalent to the following command line statement
 function rebase!(repo::GitRepo, upstream::AbstractString="", newbase::AbstractString="")
     with(head(repo)) do head_ref
         head_ann = GitAnnotated(repo, head_ref)
-        upst_ann  = if isempty(upstream)
+        upst_ann = if isempty(upstream)
             brn_ref = LibGit2.upstream(head_ref)
-            if brn_ref === nothing
+            if isnull(brn_ref)
                 throw(GitError(Error.Rebase, Error.ERROR,
                                "There is no tracking information for the current branch."))
             end
             try
-                GitAnnotated(repo, brn_ref)
+                GitAnnotated(repo, get(brn_ref))
```

Comment:
There are a lot of other places where you've used `Base.get` for nullable unpacking here though - does the LibGit2 module currently import `Base.get` ? If so it's extending a different concept in a way it probably shouldn't be. If not, then this line is calling the wrong get